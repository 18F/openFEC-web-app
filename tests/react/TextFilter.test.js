const rendered = require('react-test-renderer');
const React = require('react');
const ReactDOM = require('react-dom');
const TextFilter = require('../../static/js/legal/filters/TextFilter');

it('renders dropdown', () => {
  const requestorOptions = [{value: 'opt1', text: 'Option 1'}, {value: 'opt2', text: 'Option 2'}];
  const component = rendered.create(
    <TextFilter key="ao_no" name="ao_no" label="AO number" value=""
        handleChange={() => {}} getResults={() => {}} />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
})
