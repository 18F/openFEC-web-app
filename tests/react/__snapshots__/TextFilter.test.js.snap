// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders dropdown 1`] = `
<div
  className="filter"
>
  <label
    className="label"
    htmlFor="ao_no-filter"
  >
    AO number
  </label>
  <div
    className="combo combo--search--mini"
  >
    <input
      className="combo__input"
      id="ao_no-filter"
      name="ao_no"
      onChange={[Function]}
      type="text"
      value=""
    />
    <button
      className="combo__button button--search button--standard"
      onClick={[Function]}
    >
      <span
        className="u-visually-hidden"
      >
        Search
      </span>
    </button>
  </div>
</div>
`;
